api-2.2/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirConditionDaoImpl_2_2Test.java

```diff
@@ -106,6 +267,7 @@ public void shouldSaveNewCondition() {
 		assertThat(result.getUuid(), equalTo(CONDITION_UUID));
 	}
 	
+	/* TODO comment out and fix before submit, once the conflicts in the initial_data.xml file are resolved.
```

Comment:
I will update this unit-test later (before merge); the problem is that the main input data file (`FhirConditionDaoImplTest_initial_data.xml`) has been changed while I was working on this PR and the change is fairly significant. I also needed to make changes in that file, so I need to figure out how the new save feature works and why this test fails with my version of the input data file.

Reply:
This is resolved now.